packages/presentation/presentation_publishing/interfaces/publishes_filters.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { Filter } from '@kbn/es-query';
+import { PublishingSubject } from '../publishing_subject';
+
+export interface PublishesFilters {
```

Comment:
Why a new interface entirely here rather than splitting out some aspects of `publishes_unified_search`?

Reply:
It felt cleaner to me than doing a `Pick` on `PublishesUnifiedSearch`... Moved the definition of this interface to the `publishes_unified_search` file in bce478bc5cd1380ac7176e4ead67f980adaca230 though, since I think you're right and they do belong together :+1: